sdk/eventhubs/azure-messaging-eventhubs/inc/azure/messaging/eventhubs/models/partition_client_models.hpp

```diff
@@ -35,7 +37,7 @@ namespace Azure { namespace Messaging { namespace EventHubs { namespace Models {
      * SequenceNumber or EnqueuedTime will be included (true) or excluded
      * (false).
      */
-    bool Inclusive;
+    bool Inclusive{false};
```

Comment:
nit: Mention default value in doc comment (although I imagine we want most bools to explicitly be set to false).